src/devices/Pn532/Pn532.cs

```diff
@@ -1588,43 +1957,68 @@ private int ReadResponseI2C(CommandSet commandSet, Span<byte> readData)
                     length += preamb[idxPreamb++];
                     length2 = preamb[idxPreamb++];
                     if ((byte)(length + length2) != 0)
+                    {
                         return -1;
+                    }
                 }
                 else
+                {
                     return -1;
+                }
             }
+
             // Is it a device to cloud message?
             if (preamb[idxPreamb++] != ToHostCheckSumD5)
+            {
                 return -1;
-            // The response command should be +1 vs the one sent            
+            }
+
+            // The response command should be +1 vs the one sent
             if (preamb[idxPreamb++] != ((byte)commandSet + 1))
+            {
                 return -1;
+            }
+
             // Finally, we can read the data
             if (length - 2 > 0)
             {
-                //var buff = new byte[length - 2];
-                //_i2cDevice.Read(buff);
-                //buff.AsSpan().CopyTo(readData);
+                ////var buff = new byte[length - 2];
```

Comment:
?

Reply:
Four slashes are used to comment out code (since two slashes alone would trigger the "add a space after //" message). Should we rather remove the commented out code completely? Undocumented inactive code is confusing, especially since the above comment suggests that `Read()` is what we want to do here.

Reply:
yes, we should remove it except for cases where it's showcasing something but I don't think that's the case here